core/repository/consensus/merkle_transaction_repository.hpp

```diff
@@ -8,9 +8,13 @@
 #include "../../consensus/consensus_event.hpp"
 
 namespace merkle_transaction_repository {
-//bool commit(std::string const hash, std::unique_ptr<consensus_event::ConsensusEvent> const tx);
-bool commit(std::string, const std::unique_ptr<consensus_event::ConsensusEvent> &event);
-std::unique_ptr<abstract_transaction::AbstractTransaction> findLeaf(std::string const hash);
+
+bool commit(std::string, const std::unique_ptr<consensus_event::ConsensusEvent> &event){
+    return false;
+}
+std::unique_ptr<abstract_transaction::AbstractTransaction> findLeaf(std::string const hash){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'hash' should be passed by reference.](https://www.codacy.com/app/soramitsu/iroha/file/4274201896/issues/source?bid=3712501&fileBranchId=3837061#l15)